src/pages/LogInWithShortLivedTokenPage.js

```diff
@@ -118,7 +121,4 @@ export default withOnyx({
     session: {
         key: ONYXKEYS.SESSION,
     },
-    betas: {
-        key: ONYXKEYS.BETAS,
-    },
```

Comment:
I think removing these betas maybe broke the transition when we are signed in because `componentDidUpdate()` never runs now. It's not really obvious, but this comment here that we deleted seems like a clue: 

https://github.com/Expensify/App/pull/7902/files#diff-94f637f52b69835bc8de01012bdb6ca75a5d07ba887bc0c9e7eb00e0a17186cbL67

This flow is so painful... we should just re-write it